src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/BulkScanAutoUpdateTest.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.hmcts.reform.bulkscan.endtoendtests;
+
+import com.google.common.collect.ImmutableMap;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.bulkscan.endtoendtests.client.CcdClient;
+import uk.gov.hmcts.reform.bulkscan.endtoendtests.helper.Await;
+import uk.gov.hmcts.reform.bulkscan.endtoendtests.helper.StorageHelper;
+import uk.gov.hmcts.reform.bulkscan.endtoendtests.helper.ZipFileHelper;
+
+import java.util.List;
+import java.util.Map;
+
+import static java.util.stream.Collectors.toMap;
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.bulkscan.endtoendtests.helper.Container.BULKSCAN_AUTO;
+import static uk.gov.hmcts.reform.bulkscan.endtoendtests.utils.ProcessorEnvelopeStatusChecker.getZipFileStatus;
+
+public class BulkScanAutoUpdateTest {
+
+    @Test
+    public void should_dispatch_blob_and_create_exception_record_for_classification()
+        throws Exception {
+
+        var zipArchive = ZipFileHelper.createZipArchive("test-data/new_application", BULKSCAN_AUTO);
+
+        StorageHelper.uploadZipFile(BULKSCAN_AUTO, zipArchive);
+
+        Await.envelopeDispatched(zipArchive.fileName);
+        Await.envelopeCompleted(zipArchive.fileName);
+
+        //get the process result again and assert
+        assertCompletedProcessorResult(zipArchive.fileName);
+
+        String ccdId = getZipFileStatus(zipArchive.fileName).get().ccdId;
+
+        Map<String, Object> caseData =
+            CcdClient.getCaseData(ccdId, BULKSCAN_AUTO.idamUserName, BULKSCAN_AUTO.idamPassword);
+        Map<String, String> ocrData = getOcrData(caseData);
+        assertThat(ocrData.get("first_name")).isEqualTo("Name");
```

Comment:
case first value is name and surname , we should change them and check update is working?

Reply:
I will do this in the next PR.